uiowa/uiowa
# [Registrar] Department codes

Resolves https://github.com/uiowa/uiowa/issues/8122. 



# How to test
```
ddev blt ds --site=registrar.uiowa.edu && ddev drush @registrar.local uli /node/256/layout
```
1. Add "Department codes" block to the page and save.
2. Confirm that `naturalKey`, `itchCode` , `description`, `status` display in a table as "Subject", "Legacy Code" ,  "Description", and "Status".
3. Confirm that you can search the table for results.
4. Test for accessibility
